src/api/app/assets/javascripts/webui2/tokenfield.js

```diff
@@ -0,0 +1,381 @@
+/* jQuery Tags Input Revisited Plugin
```

Comment:
This file should go to [vendor/assets](https://guides.rubyonrails.org/asset_pipeline.html#asset-organization)

Reply:
I was wondering if we had something for this. Thank you for the pointer!

Reply:
Thinking about this, I think this should also be done for CodeMirror (so {javascripts,stylesheets}/cm2  and collapse-text.scss files).